app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
> I think that's browser-dependent?

Maybe configuration dependent? I tested it on OSX (FF, Chrome, Safari), Windows (IE, Chrome) and Linux (FF, Chrome) and the middle button always opened the link in a new tab.

> To @cantino's point about a confirm, feels like there should be a confirm dialog any time you navigate away from unsaved agent changes. But perhaps that's a separate issue?

I agree, additionally if we do it for the agent form we also should be consistent and add it to every form we have. However generally I am not a huge fan of confirm popups in that case :)